consensus/istanbul/celo-ibft-spec.md

```diff
@@ -448,3 +448,29 @@ function call to occur after the given duration.
 // to Hc and value equal to V is greater than 1 and less than 10.
 1 < |{ m<Prepare, Hc, Rm, Vm> ∈ M : Rm = Hc && Vm = V }| < 10
 ```
+
+## Appendix 2: Notes
+
+### What do the prepared certificate and round change certificate do?
+
+They enforce that if it’s possible that a node could have confirmed a block in
+a round, then all remaining nodes in future rounds can only confirm the same
+block.
+
+- A node could have confirmed a block if 2f+1 nodes send commit messages for a
```

Comment:
sen**d** or sen**t**?

Reply:
Yes, I think sent is actually correct, thanks! :)